packages/manager/modules/pci/src/projects/project/analytics-data-platform/deploy/storage/storage.html

```diff
@@ -38,19 +38,13 @@ <h4><strong>{{ ::'analytics_data_platform_deploy_storage_cluster_title' | transl
                 <dd class="storage-list__definition"><span data-ng-bind="::$ctrl.nodesConfig.worker.count"></span></dd>
                 <dt class="storage-list__term"><span data-translate="analytics_data_platform_deploy_storage_per_node"></span> : </dt>
                 <dd class="storage-list__definition">
-                    <span>
-                        {{ ($ctrl.data.hdfsEffectiveStorage * $ctrl.selectedCapability.hdfsReplicationFactor) / $ctrl.nodesConfig.worker.count }}
-                    </span>
-                    <span data-translate="analytics_data_platform_common_unit_gb"></span>
+                    <span data-ng-bind="$ctrl.getStoragePerNode()"></span>
```

Comment:
The `ngBind` directive could be applied directly on the `<dd>` parent element, right?

Reply:
Yes... This has been fixed. Thanks :)